src/jailer/src/env.rs

```diff
@@ -86,6 +97,16 @@ fn clone(child_stack: *mut libc::c_void, flags: libc::c_int) -> Result<libc::c_i
     .map_err(JailerError::Clone);
 }
 
+#[derive(Debug, thiserror::Error)]
+enum UserfaultfdParseError {
+    #[error("Could not read /proc/misc: {0}")]
+    ReadProcMisc(#[from] std::io::Error),
+    #[error("Could not parse minor number: {0}")]
+    ParseDevMinor(#[from] std::num::ParseIntError),
+    #[error("/dev/userfaultfd device not loaded")]
```

Comment:
nit: we don't check for the device node, so maybe remove `/dev/` to avoid possible confusion for someone trying to create the device on their own via mknod to fix the error.